docs_builder/requirements-docs.txt

```diff
@@ -4,4 +4,4 @@ nbsphinx>=0.8.1
 sphinx_rtd_theme>=0.5.1
 pypandoc>=1.4
 jupyterlab>=3.0.5
-
+sphinx-autoapi>=1.7.0
```

Comment:
always use the latest dependencies. We try to enforce the latest versions as much as possible.
I will update this.

Reply:
I understand the situation, I think I had just added the sphinx-autoapi version (i got it from [pypi.org](https://pypi.org/project/sphinx-autoapi/) webpage) and it had slipped my mind to update the remaining dependencies. I hope this is what we are looking to update here. If you say, I can make an update for the all the dependencies:

- ```pypandoc``` to 1.5
- ```jupyterlab``` to 3.0.7

 The latest version of ```sphinx-autoapi``` is shown as 1.7.0, should we change it to some other version? Shall I go ahead and make these changes to the dependencies?